### ISSUE TITLE
About the Muti-class U-Net

### ISSUE THREAD

**Issue by GuoQingqing** [DengPingFan/Inf-Net#30]
Hi! 
 You have done a splendid work, and I have a question about the loss and label  about Muti-class U-Net. Why don't you  just use the CrossEntropyLoss instead of the BCELoss and one-hot label? 
It would be very kind of you to answer my question.
Thanks.

**Comment by DengPingFan** [comment]
Hi, 

Thanks for your attention. Because we mainly focus on the semi-supervised learning framework. Actually, there are also other training losses that can be used such as Hybrid-Eloss ([link](https://github.com/GewelsJI/Hybrid-Eloss)). You can try other losses also.

Best,
